common4j/src/main/com/microsoft/identity/common/java/exception/ErrorStrings.java

```diff
@@ -503,4 +509,9 @@ private ErrorStrings() {
      * Use this message for when Device Code Flow fails with an error code that doesn't match any of the pre-defined Device Code Flow codes.
      */
     public final static String DEVICE_CODE_FLOW_DEFAULT_ERROR_MESSAGE = "Device Code Flow has failed with an unexpected error. The error code shown was received from the result object.";
+
+    /**
+     * Access token doesn't exist and is required for NativeAuth token refresh
+     */
+    public static final String NATIVE_AUTH_NO_ACCESS_TOKEN_FOUND = "native_auth_no_access_token_found";
```

Comment:
```suggestion
    public static final String NATIVE_AUTH_NO_ACCESS_TOKEN_FOUND_ERROR_CODE = "native_auth_no_access_token_found";
```
Do we not need an error message as well?

Reply:
Done